framework/include/loops/ThreadedElementLoopBase.h

```diff
@@ -293,15 +302,17 @@ template <typename RangeType>
 void
 ThreadedElementLoopBase<RangeType>::preBoundary(const Elem * /*elem*/,
                                                 unsigned int /*side*/,
-                                                BoundaryID /*bnd_id*/)
+                                                BoundaryID /*bnd_id*/,
+                                                const Elem * /* nullptr*/)
 {
 }
 
 template <typename RangeType>
 void
 ThreadedElementLoopBase<RangeType>::onBoundary(const Elem * /*elem*/,
                                                unsigned int /*side*/,
-                                               BoundaryID /*bnd_id*/)
+                                               BoundaryID /*bnd_id*/,
+                                               const Elem * /*=nullptr*/)
```

Comment:
The other commented code in this signature are names instead of values. I would say either do nullptr -> lower_d_elem or nullptr -> `lower_d_elem = nullptr`